cookbook/lambda_magic.rst

```diff
@@ -400,6 +400,77 @@ will return ``NaN``, which corresponds to ``unknown`` sensor state.
         name: "Number from text"
 
 
+Factory reset after 5 quick reboots
+-----------------------------------
+
+One may want to restore factory settings (eg. Wi-Fi credentials set at runtime, or clear restore states) without having to
+disassemble or dismount the devices from their deployed location, whilst there's no network access either. The example below
+shows how to achieve that using lambdas in a script by triggerig the factory reset switch after the system rebooted 5 times
+with 10 seconds timeframes.
+
+.. code-block:: yaml
+
+    # Example config.yaml
+    esphome:
+      name: "esphome_ld2410"
+      on_boot:
+        priority: 600.0
+        then:
+          - script.execute: fast_boot_factory_reset_script
+    esp32:
+      board: esp32-c3-devkitm-1
+    
+    globals:
+      - id: fast_boot
+        type: int
+        restore_value: yes
+        initial_value: '0'
+    
+      - id: factory_reset_boot_count_trigger
+        type: int
+        initial_value: '5'
```

Comment:
This doesn't really make sense as a global, it could just be a substitution

Reply:
Done